### PR TITLE
[11.0.X] Update rivet-toolfile.spec - Try 5521 on 11_0_master

### PR DESCRIPTION
**Pull request by mrodozov**
please test

### CODE REVIEW

**Comment by mrodozov** [comment]
@intrepid42 I copied it here

File: rivet-toolfile.spec [hunk 1]
```diff
@@ -16,6 +16,8 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/rivet.xml
 <environment name="INCLUDE" default="$RIVET_BASE/include"/>
 </client>
 <runtime name="PATH" value="$RIVET_BASE/bin" type="path"/>
+<runtime name="RIVET_ANALYSIS_PATH" value="$RIVET_BASE/lib/Rivet" type="path"/>
+<runtime name="RIVET_DATA_PATH" value="$RIVET_BASE/share/Rivet" type="path"/>
 <runtime name="RIVET_ANALYSIS_PATH" value="$RIVET_BASE/lib" type="path"/>
```
Comment (smuzaffar) [inline rivet-toolfile.spec]: I think this line was deleted in #5520 ... right?

Comment (mseidel42) [inline rivet-toolfile.spec]:
```suggestion

```
This line needs to be deleted

**Comment by mrodozov** [comment]
please test

File: rivet-toolfile.spec (hunk 1 above)
Comment (mrodozov) [reply rivet-toolfile.spec]: it did I missed it

**Review by mseidel42 — APPROVED** [review]

**Comment by smuzaffar** [comment]
+externals

**Comment by silviodonato** [comment]
backport of #5520

**Comment by silviodonato** [comment]
+1

**Pull request merged**